components/consumers/defectdojo/main.go

```diff
@@ -27,6 +30,14 @@ var (
 	issueTemplate          string
 )
 
+func getEngagementTime(engagementTime *timestamppb.Timestamp, scanID string) string {
+	if time.Time.IsZero(engagementTime.AsTime()) {
```

Comment:
Should this be done on `engagementTime` directly?

`engagementTime.AsTime().IsZero()`